AltinnCore.sln

```diff
@@ -37,6 +38,8 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "AltinnCore.Authentication",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "PrefillClient", "src\Altinn.ExampleClients\PrefillClient\PrefillClient.csproj", "{7D5C7252-B316-4DC6-B7E7-18613DAB367C}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Storage.Interface", "src\Altinn.Platform\Altinn.Platform.Storage\Storage.Interface\Storage.Interface.csproj", "{D9F6DAB7-E921-44B6-A1A6-5796DCAE0D07}"
```

Comment:
What is the background for including storage.interface in the altinnCore solution?